docs/disclaimer.md

```diff
@@ -0,0 +1,5 @@
+# Disclaimer
+
+The Knowledge Base, including all the Help articles on this site, is provided for technical support purposes only, without representation, warranty or guarantee of any kind. Not an offer to sell or solicitation of an offer to buy any security or other regulated financial instrument. Not technical, investment, financial, accounting, tax, legal or other advice; please consult your own professionals. Please conduct your own research before connecting to or interacting with any dapp or third party or making any investment or financial decisions. MoonPay, ParaSwap and any other third party services or dapps you access are offered by third parties unaffiliated with us. Please review this [Notice](https://assets.website-files.com/602e8e4411398ca20cfcafd3/60ec9607c853cd466383f1ad_Important%20Notice%20-%20avalabs.org.pdf) and the [Terms of Use](https://wallet.avax.network/legal).
+
```

Comment:
This [Terms of Use](https://wallet.avax.network/legal) doesn't load when you click from an outside domain, such as this PR. Instead it redirects to [the landing page](https://wallet.avax.network/access). Only when I click the `Terms Of Use` link on the Wallet's homepage does the `Terms of Use` load.

Reply:
This looks like fixed now.